### PR TITLE
March into Mapness - Monkey Temple

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Adds a map for marping marpch :)

### Monkey Temple

*"Honestly, when a prospector told me they found a man-made structure inside of a random asteroid, I laughed."*

Adds a new map. A derelict shuttle attached to an asteroid, with a temple inside.

temple

### Monkey Paw

*The monkey paw curls, granting your wish... but at what cost?*

A reward of the monkey temple. It grants wishes. You get what you want, but the wish is twisted. Also drops VERY rarely from gibbing monkeys.

### Mapping March
Ckey to recieve rewards: armhulenn

## Why It's Good For The Game

Should be fun to explore! More spess variety makes spess less of a "find that one ruin" and more "what will I find out there?"

## Changelog
:cl: Armhulen code, Brute sprites
add: New space ruin, a derelict with a temple.
add: Monkey Paw, and Monkey Paw wishes
/:cl:

### CODE REVIEW

File: code/modules/requests/request_manager.dm [hunk 1]
```diff
@@ -67,6 +69,17 @@ GLOBAL_DATUM_INIT(requests, /datum/request_manager, new)
...
 			SEND_SOUND(admin, sound('sound/effects/pray.ogg'))
 
+/**
+ * Creates a request for a monkey paw wish
+ *
+ * Arguments:
+ * * C - The client who is sending the request
+ * * message - The message
+ */
+/datum/request_manager/proc/monkey_paw_wish(client/C, message)
```
Comment (Fikou) [inline code/modules/requests/request_manager.dm:79]: dont use single letter variable names

**Comment by LordVollkorn** [comment]
The monkey paw is a pretty cool idea and since negative side effects are implied, admins could be more willing to grant the wishes than if they came via prayer

**Comment by san7890** [comment]
you must add this to https://github.com/tgstation/tgstation/blob/master/config/spaceruinblacklist.txt

**Comment by tralezab** [comment]
SO I WAS A LITTLE BUSY

**Comment by Rex9001** [comment]
Tragic.

**Pull request closed without merging**